phpBB/phpbb/notification/type/base.php

```diff
@@ -162,12 +162,12 @@ public function __toString()
 	/**
 	* Get special data (only important for the classes that extend this)
 	*
-	* @param string $name Name of the variable to get
+	* @param string|false $name Name of the variable to get, false if all data should be returned
```

Comment:
string|null or ?? false below

And also maybe array?, because I see that `$this->data['notification_data']` behaves as an array and if $name=false, it return it